.gitignore

```diff
@@ -1,4 +1,43 @@
-/vendor/*
+# CakePHP specific files #
+##########################
 /config/app.php
-/tmp/*
+/config/.env # josegonzalez/php-dotenv
 /logs/*
+/tmp/*
+/tmp/bake/*
```

Comment:
if tmp is already in, please do not add also all subfolders as well.
Lets keep it simple when adding this again please.